src/internal/time.rs

```diff
@@ -6,7 +6,7 @@ use std::time::{Duration, SystemTime, UNIX_EPOCH};
 pub fn filetime_from_system_time(timestamp: SystemTime) -> u64 {
     match timestamp.duration_since(filetime_epoch()) {
         Ok(delta) => {
-            delta.as_secs() * 10_000_000 + (delta.subsec_nanos() / 100) as u64
+            delta.as_secs() * 1_000_000_000 + delta.subsec_nanos() as u64
```

Comment:
Why these changes?

Reply:
Yes, because the tests are falling: https://github.com/ikrivosheev/rust-msi/runs/5552772525?check_suite_focus=true#step:4:207

`nanos` is 10^(-9)

Reply:
I can rewrite tests and revert this changes?

Reply:
Ah, I think I understand.

Windows [FILETIME](https://docs.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-filetime)s are measured in 10-million-ths of a second, that is, in 100-nanosecond intervals.  So calculating the filetime as `delta.as_secs() * 10_000_000 + (delta.subsec_nanos() / 100)` is intentional.

Looks like the failing test is checking that `PropertyValue::FileTime(SystemTime::now())` can round-trip, and that's failing on (ubuntu-20.04, stable) with:
```
---- internal::propset::tests::property_value_round_trip stdout ----
thread 'internal::propset::tests::property_value_round_trip' panicked at 'assertion failed: `(left == right)`
  left: `FileTime(SystemTime { tv_sec: 1647343851, tv_nsec: 381626500 })`,
 right: `FileTime(SystemTime { tv_sec: 1647343851, tv_nsec: 381626523 })`', src/internal/propset.rs:508:13
```

I think the problem is that a Windows `FILETIME` can only represent a 100-nanosecond resolution, but the `SystemTime` clock on that OS has a higher resolution than that, so of course it can't always round-trip perfectly.  So the test itself is flawed.

Instead of using `SystemTime::now()` and expecting it to round-trip perfectly, I think we should either be using some specific `SystemTime` that is a multiple of 100 nanoseconds since the epoch, or else writing a different test that makes sure that the `PropertyValue::FileTime` round-trips to within 100 nanoseconds.

Reply:
Done